release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -71,8 +71,9 @@ If you are creating machine configs for day 1 or day 2 operations that use Ignit
 [id="ocp-4-6-additional-steps-to-add-nodes-to-clusters"]
 ==== Additional steps to add nodes to existing clusters
 
-Because of the changes to the Ignition specification, if you want to add more nodes to your {product-title} cluster by using the latest version of {op-system} boot media, you might need to manually modify your Ignition configuration as part of the process.
-//forthcoming KCS links
+On clusters that have been upgraded to 4.6, because of the changes to the Ignition specification, if you want to add more nodes to your {product-title} cluster by using the latest version of {op-system} boot media, you might need to manually modify your Ignition configuration as part of the process.
```

Comment:
The sentence is a little long now (with 4 sections/3 commas). Not sure if it's possible to break it up at all, but I'd at least take a look. 

Also, maybe "On" -> "For" ?